### ISSUE TITLE
Plotting Model Accuracy/Loss Plots

### ISSUE THREAD

**Issue by katreparitosh** [amaiya/ktrain#221]
Hello,

In Keras, we can access the "History" object and plot the model accuracy/loss plots to assess how it progresses. These plots help us infer the overfitting/underfitting aspects of the model. 

Similarly, is there a way in Ktrain where we can plot the training history in terms of Loss/Accuracy? I would be grateful if you could guide me. 

Ref: https://machinelearningmastery.com/display-deep-learning-model-training-history-in-keras/ 

Please see the ref article.

Thanks

**Comment by amaiya** [comment]
This is answered in [the FAQ](https://github.com/amaiya/ktrain/blob/master/FAQ.md#how-do-i-retrieve-or-visualize-training-history).

All `*fit*` method in *ktrain* return training history similar to Keras.  *ktrain* also has a convenience method to plot training vs. validation loss automatically: `learner.plot('loss')`.  This is explained in [the FAQ]()

**Comment by katreparitosh** [comment]
Thank you so much.
